README.md

```diff
@@ -31,6 +31,12 @@ Simply install the package through `npm`
 npm i ng-material-treetable --save
 ```
 
+Make sure you have the angular material packages installed
+
+```
+npm install @angular/material @angular/cdk @angular/animations --save
```

Comment:
It'd be good to be consistent and keep using the short version of `npm install` -> `npm i`

Reply:
Oh yes, totally missed that.